client/src/pages/Login/Login.tsx

```diff
@@ -39,22 +38,17 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid className={classes.paper} item component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center" component={Paper}>
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
-              </Grid>
-            </Grid>
-            <LoginForm handleSubmit={handleSubmit} />
-          </Box>
-          <Box p={1} alignSelf="center" />
-        </Box>
-      </Grid>
+      <NavBar />
```

Comment:
Is the `<NavBar>` component going to appear for all pages? If so, instead of adding it to every single page, you could move it outside of the routes - ideally as a part of the layout if you have one.

Reply:
TODO: I Will implement a layout page.